### PR TITLE
generate partials in docs

### PR DESCRIPTION
**Pull request by hidalgopl**
This is CI workflow that will be triggered on the tag push (`vcluster-v*` or `platform-v*`). It is expected to generate partials in the vcluster-docs repository and open a PR there.
More context: https://www.notion.so/loftsh/vCluster-jsonschema-CI-design-doc-14210940806980b5ac2cfb4b481d5e4b?d=14310940806980589404001c653d4b48#1421094080698000ae75db8e5d45f833

part of ENG-3393

### CODE REVIEW

File: .github/workflows/update-vcluster-docs.yaml [hunk 1]
```diff
@@ -0,0 +1,109 @@
+name: Update partials in documentation
```
Comment (Piotr1215) [inline .github/workflows/update-vcluster-docs.yaml:1]: Can we add `actor` name triggering the event? It's optional, but helpful when looking at the `Actions` tab.

```yaml
run-name: ${{ github.actor }} is updating partials
```

File: .github/workflows/update-vcluster-docs.yaml [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+          if git diff-index --quiet HEAD --; then
+            exit 0
+          fi
+
+          # set git info
+          git config --global user.name "Loft Bot"
+          git config --global user.email 'loft-bot@users.noreply.github.com'
+          echo "Changes detected"
+
+          # commit changes
```
Comment (Piotr1215) [inline .github/workflows/update-vcluster-docs.yaml:66]: Please consider swapping to a [dedicated action](https://github.com/peter-evans/create-pull-request).

**Review by Piotr1215 — COMMENTED** [review]
Thanks @hidalgopl , left a few improvement suggestions.

File: .github/workflows/update-vcluster-docs.yaml (hunk 1 above)
Comment (hidalgopl) [reply .github/workflows/update-vcluster-docs.yaml:1]: that's a good idea, I wasn't aware of `github.actor` :+1

File: .github/workflows/update-vcluster-docs.yaml [hunk 3]
```diff
@@ -0,0 +1,109 @@
+name: Update partials in documentation
+
+on:
```
[the remark this answers is not in the document]
Comment (hidalgopl) [reply .github/workflows/update-vcluster-docs.yaml:7]: added

File: .github/workflows/update-vcluster-docs.yaml [hunk 4]
```diff
@@ -0,0 +1,109 @@
...
+        description: 'Release tag in form vcluster-vX.Y or platform-vX.'
+        required: true
+        type: string
+
+jobs:
+  sync:
+    runs-on: ubuntu-latest
+    outputs:
+      version_tag: ${{ steps.parseVersion.outputs.versionTag }}
+    steps:
```
[the remark this answers is not in the document]
Comment (hidalgopl) [reply .github/workflows/update-vcluster-docs.yaml:24]: moved cloning the repo up and added with `go-version-file`

File: .github/workflows/update-vcluster-docs.yaml (hunk 2 above)
Comment (hidalgopl) [reply .github/workflows/update-vcluster-docs.yaml:74]: I can't find a support there to create a PR in different repository, so I'll stick with `gh pr create ...`

**Review by Piotr1215 — APPROVED** [review]
Thank you for the changes @hidalgopl , LGTM!

**Pull request merged**